project/Dependencies.scala

```diff
@@ -67,8 +67,8 @@ object Dependencies {
       "ch.qos.logback" % "logback-classic" % logbackVersion,
       "com.typesafe.scala-logging" %% "scala-logging" % loggingVersion
     )
-    val xgboostDep = "ml.dmlc" % "xgboost4j" % xgboostVersion
-    val xgboostSparkDep = "ml.dmlc" % "xgboost4j-spark" % xgboostVersion
+    val xgboostDep = "ml.dmlc" % "xgboost4j" % xgboostVersion // scala 2.11 only
```

Comment:
For the record: xgboost master is 2.12 only: https://github.com/dmlc/xgboost/pull/4574

Reply:
@mengxr Is the comment `// scala 2.11 only` used here as a hint to tell SBT only use scala-2.11 when compiling with this dependency ?